File: 007/2023_11_24/abc113b.jl
N = parse(Int, readline())
T, A = broadcast(n -> parse(Int, n), split(readline()))
Hn = broadcast(n -> parse(Int, n), split(readline()))

answer = 1
templature = abs(A * - T * - Hn[1] * 0.006)

for i in 2:length(Hn)
    current_templature = abs(A * - T * - Hn[i] * 0.006)

    if current_templature ≤ templature
        global answer = i
        current_templature = templature
    end
end

println(answer)

File: 01/002/2023_10_26/abc113b.py
N = int(input())
T, A = map(int, input().split())
Hn = list(map(int, input().split()))

answer = 0
currentAbs = 1000

for i in range(N):
    templature = abs(A - T - Hn[i] * 0.006)

    if currentAbs >= templature:
        currentAbs = templature
        answer = i + 1

print(answer)
